docs/UserGuide.adoc

```diff
@@ -241,6 +241,45 @@ Format: `history`
 Pressing the kbd:[&uarr;] and kbd:[&darr;] arrows will display the previous and next input respectively in the command box.
 ====
 
+=== Check your email : `check`
+
+Checks whether you have set your email or not, and displays the censored email address if you have set it before. +
+Format: `check`
```

Comment:
Good that you have changed `check` to `checkemail` in the Dev Guide, but the User Guide isn't updated.

Reply:
Fixed.